src/txmempool.h

```diff
@@ -794,19 +802,28 @@ class CTxMemPool
     /** UpdateForDescendants is used by UpdateTransactionsFromBlock to update
      *  the descendants for a single transaction that has been added to the
      *  mempool but may have child transactions in the mempool, eg during a
-     *  chain reorg.  setExclude is the set of descendant transactions in the
-     *  mempool that must not be accounted for (because any descendants in
-     *  setExclude were added to the mempool after the transaction being
-     *  updated and hence their state is already reflected in the parent
-     *  state).
+     *  chain reorg.
      *
-     *  cachedDescendants will be updated with the descendants of the transaction
-     *  being updated, so that future invocations don't need to walk the
-     *  same transaction again, if encountered in another transaction chain.
+     * @param[in] updateIt the entry to update for it's descendants
+     * @param[in] cachedDescendants a cache where each line corresponds to all
+     *     descendants. It will be updated with the descendants of the transaction
+     *     being updated, so that future invocations don't need to walk the same
+     *     transaction again, if encountered in another transaction chain.
+     * @param[in] setExclude the set of descendant transactions in the mempool
+     *     that must not be accounted for (because any descendants in setExclude
+     *     were added to the mempool after the transaction being updated and hence
+     *     their state is already reflected in the parent state).
+     * @param[out] descendants_to_remove Populated with the txids of entries that
+     *     exceed ancestor limits. It's the responsibility of the caller to
+     *     removeRecursive them.
+     * @param[in] ancestor_size_limit the max number of ancestral bytes allowed
+     *     for any descendant
+     * @param[in] ancestor_count_limit the max number of ancestor transactions
+     *     allowed for any descendant
```

Comment:
Limits are inclusive and apply to all entries

```suggestion
     * @param[in] ancestor_size_limit      The maximum allowed size in virtual bytes of an entry and its ancestors
     * @param[in] ancestor_count_limit     The maximum allowed number of transactions including the entry and its ancestors.

```

Reply:
fwiw if you use a text editor which understands doxygen you get something which looks fine without a visual gap.
<img width="588" alt="image" src="https://user-images.githubusercontent.com/886523/132998517-30b6dee2-137e-46c6-8a26-307a5cdea93c.png">